web/src/main/ui/app/components/flows-new/edit-flow/mastering/matching/ui/add-match-threshold-dialog.component.ts

```diff
@@ -16,6 +17,7 @@ export interface DialogData {
 })
 export class AddMatchThresholdDialogComponent {
 
+  instantErrorMatcher: InstantErrorStateMatcher;
```

Comment:
you can define and instantiate the 'instantErrorMatcher' all here and not do it in the OnInit method.
instantErrorMatcher: InstantErrorStateMatcher = new InstantErrorStateMatcher();